independent-publisher-connectors/OMDb API/README.MD

```diff
@@ -0,0 +1,36 @@
+## The Open Movie Database
+The OMDb API is a RESTful web service to obtain movie information, all content and images on the site are contributed and maintained by users.
+
+## Publisher
+Aaryan Arora
+
+## Prerequisites
+You need to get the API key from [ORB Website](http://www.omdbapi.com/) or [here](http://omdbapi.com/apikey.aspx). The API Key is available publicly.
+
+### Deploying the sample
+Run the following commands and follow the prompts:
+
+```paconn
+paconn create --api-def apiDefinition.swagger.json --api-prop apiProperties.json --secret <client_secret>
+```
+
+## Supported Operations
+The connector supports the following operations:
+* `Search By Title`: List keys in the specified vault
+* `Search By ID`: Gets the public part of a stored key
+* `Search by Keyword`: Creates a new key
+
+## Parameters
+
+parameter | required  | description
+--- | --- | ---
+`API Key` | Yes | The API Key for the OMDb API
+`i` | No | A valid IMDb ID
+`s` | No | Movie title to search for
+`type` | No | Type of result to return
+`page` | No | Page number to return
+
+Please note while both "i" and "s" are optional at least one argument is required
+
+## API Documentation
+Visit [OMDb reference page](http://www.omdbapi.com/) for further details.
```

Comment:
Hi @aaryan2134, 
Thank you so much for submitting your connector proposal.
I wanted to bring your attention to some points you may need to fix.
In the readme file you are missing the Known Issues and Limitations section. if there are no issues known yet, you can just mention that No Issues and Limitations yet.

Reply:
Yeah sure. Done